doc/index.md

```diff
@@ -358,3 +117,19 @@ The `hello-ubuntu.json` file referenced above could contain something like:
     }
 }
 ```
+
+
+
+```{toctree}
```

Comment:
This toctree will be visible in the current documentation (but it is at the bottom, so probably not a big deal).